### PR TITLE
ci: add dependabot config

### PR DESCRIPTION
**Pull request by maxmoehl**

### CODE REVIEW

File: ci/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Fixes
+- fix a
```
Comment (plowin) [inline ci/release_notes.md:2]: Please remove the fix-lines, upgrades and thank yourself :)

File: ci/release_notes.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- fix b
+
+# New Features
+- introduced @dependabot to automatically bump our ci dependencies (@maxmoehl)
```
Comment (plowin) [inline ci/release_notes.md:6]:
```suggestion
- Introduce @dependabot to automatically bump our ci dependencies (@maxmoehl)
```

**Review by plowin — APPROVED** [review]
Thx!
Minor remarks

File: ci/release_notes.md (hunk 1 above)
Comment (maxmoehl) [reply ci/release_notes.md:2]: I always find it weird to thank myself. GitHub shows the contributors anyways so maybe a line like `Huge thanks to all contributors!` would be more appropriate? I will add myself this time but maybe we can change this in the future.

**Pull request merged**